src/core_plugins/metrics/public/components/__tests__/add_delete_buttons.js

```diff
@@ -0,0 +1,67 @@
+// import React from 'react';
+// import { expect } from 'chai';
+// import { shallow } from 'enzyme';
+// import sinon from 'sinon';
+// import AddDeleteButtons from '../add_delete_buttons';
+// import Tooltip from '../tooltip';
+
+// describe('<AddDeleteButtons />', () => {
```

Comment:
I agree with your sentiment but we (team) wanted to get this commit in this week so it would be available via snapshots for Elastic{ON} and allow more time for people to use it before 5.4 ships. Luckily this is "soft"ware and we can work towards getting Enzyme working again. It broke when the Babel upgrade occurred; had this been merged before that upgrade it would hve been addressed by that team. The issue I have with keeping PR's  like this separate for too long is you keep having to chase master as they change things; the PR keeps getting bigger and never seems to finish. With this merged we can now focus on creating smaller more manageable PR's to fix anything that needs addressed between now and code freeze.